cms/src/web/profiles/open_pension/modules/open_pension_fetcher/open_pension_fetcher.routing.yml

```diff
@@ -0,0 +1,8 @@
+open_pension_fetcher.links:
+  path: '/api/fetcher-links'
+  defaults:
+    _title: 'Example'
+    _controller: '\Drupal\open_pension_fetcher\Controller\OpenPensionFetcherLinksController::build'
+  methods: ['PATCH', 'POST']
+  requirements:
+    _permission: 'access content'
```

Comment:
switch to a custom access requirements and make sure we can verify this came from our containers. Can be handled in another PR